subsys/bluetooth/controller/ll_sw/pdu.h

```diff
@@ -32,6 +32,10 @@
 
 #define BYTES2US(bytes, phy) (((bytes)<<3)/BIT((phy&0x3)>>1))
 
+/* Advertisement channel minimum payload size */
+#define PDU_AC_PAYLOAD_SIZE_MIN 1
+/* Advertisement channel minimum payload size */
+#define PDU_AC_PAYLOAD_SIZE_MIN 1
```

Comment:
Was one of these supposed to be a `MAX`?

Reply:
Hm... I've got not idea why the macro is defined here twice. I'll remove one occurrence. `MAX` is not needed.